trimmer_decorator.rb

```diff
@@ -0,0 +1,11 @@
+require './decorator'
+require './capitalize_decorator'
+require './person'
+
+# class trimmer_decorator
+class TrimmerDecorator < Decorator
+  def correct_name
+    super()
+    @nameable.length > 10 ? @nameable[0..9] : @nameable
```

Comment:
- [ ] also in this  should return a trimed correct_name method of nameable `@nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name`